demo/genome_config.yaml

```diff
@@ -0,0 +1,206 @@
+lcr-modules:
+
+    _shared:
+        lcr-modules: "../"
+        lcr-scripts: "../../lcr-scripts"
+        root_output_dir: "results/"
+        scratch_directory: "scratch/"
+        unmatched_normal_ids:
+            genome--grch37: "TCRBOA7-N-WGS"
+    
+    battenberg:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "data1/{sample_id}.bam"
+        
+        reference_path:
+            hg38: " --chr_prefixed_genome "                           # Copied from vcf2maf
+            grch37: " reference/genomes/grch37/downloads/hg19-grch37 "     # Copied from vcf2maf
```

Comment:
Are these comments meant for the users or for you? Or for code reviewers?

Reply:
Removing the whole section in the new Battenberg update

Reply:
Great